src/main/java/frc/robot/Robot.java

```diff
@@ -12,8 +12,6 @@
 
 package frc.robot;
 
-import static frc.robot.Constants.LoggerConstants.*;
```

Comment:
No removals that used this. Was this just an unused import?

Reply:
was removed